doc/source/data/transforming-data.rst

```diff
@@ -17,15 +17,16 @@ This guide shows you how to:
 * :ref:`Stateful transforms <stateful_transforms>`
 * :ref:`Groupby and transform groups <transforming_groupby>`
 * :ref:`Shuffle data <shuffling_data>`
-* :ref:`Repartition data <repartitioning_data>`
 
 .. _transforming_rows:
 
 Transforming rows
 =================
 
-To transform rows, call :meth:`~ray.data.Dataset.map` or
-:meth:`~ray.data.Dataset.flat_map`.
+.. tip::
+
+    If your transformation is vectorized, call :meth:`~ray.data.Dataset.map_batches` for
+    better performance. To learn more, see `Transforming batches <#transforming-batches>`_.
```

Comment:
the link to specific transforming batches section doesn't work, i think you'll need to use `:ref:`.